### PR TITLE
Add 'vite-plugin-transform' to  'Transformers'

### PR DESCRIPTION
**Pull request by Silksofthesoul**
> Please make sure all the requirements are satisfied, otherwise the PR could be closed without further notice.

## Checklist

- [x] Title as described.
- [x] Make sure you put things in the right category.
- [x] The description of your item should be a sentence with less than 24 words.
- [x] Omit unnecessary words already provided in the context (e.g. `for Vite`, `a Vite plugin`).
- [x] Always add your items to the end of a list.

### Plugins/Tools

- [x] The plugin/tool is working with **Vite 2.0 and onward**.
- [x] The project is Open Source.
- [x] The project follows the [Vite Plugins Conventions](https://vitejs.dev/guide/api-plugin.html#conventions).
- [ ] The repo should be at least 30 days old.
- [x] The documentation is in English.
- [x] The project is active and maintained.
- [x] The project accepts contributions.
- [x] Not a commercial product.

### Starter Templates

- [ ] The starter template is working with **Vite 2.0 and onward**.
- [ ] The documentation is in English.
- [ ] The starter template most provide enough instructions / documentation about how to start up and what's is included.
- [ ] A screenshot or a live demo should be included.
- [ ] The repo should be at least 30 days old.
- [ ] Should be differentiable from the existing starter templates. 

### Apps/Websites

- [ ] The website is available without errors and load in a reasonable amount of time.
- [ ] The website must be Open Source and showing it's using Vite intensively.
- [ ] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.

### CODE REVIEW

**Comment by Silksofthesoul** [comment]
@antfu Hello! I see you added the tag "change requested". I have little experience with creating a "Pull Request" and I'm not sure what I should do when looking at this label. Please tell me, I need to click on the "Resolve conflicts" button or will you or someone else do this operation?

File: README.md [hunk 1]
```diff
@@ -130,10 +130,15 @@ Use the "Table on Contents" menu on the top-left corner to explore the list.
 - [vite-plugin-handlebars](https://github.com/alexlafroscia/vite-plugin-handlebars) - Process HTML files with Handlebars.
 - [vite-plugin-html-config](https://github.com/ahwgs/vite-plugin-html-config) - HTML tag configuration, such as script link style meta favicon and more.
 - [vite-plugin-html-template](https://github.com/IndexXuan/vite-plugin-html-template) - HTML template for vite app, like html-webpack-plugin for webpack.
+- [vite-plugin-transform](https://github.com/Silksofthesoul/vite-plugin-transform) - Plugin resolve paths in cases where standard ways dont help, and replacement functionality.
```
Comment (Scrum) [inline README.md:133]: ## Checklist

- [ ] git: pull request template structure not broken
- [ ] git: pull request template not selected `type`
- [ ] git: pull request template not checked `Checklist`
- [ ] git: empty commit
- [ ] git: link not to repository
- [ ] git: link description contains a link to the author / third-party resource
- [ ] git: broken PR
- [ ] git: no conflicts resolved
- [ ] git: doublicate
- [ ] time: not updated for more than 6 months
- [ ] category: not in its category
- [x] lists: not the last on the list
- [ ] docs: no documentation
- [ ] docs: non-english documentation
- [ ] project: commercial product
- [ ] project: dead project
- [ ] project: does not accept donat
- [ ] site: no link to the site
- [ ] site: site with errors
- [ ] site: not actively using vuejs
- [ ] site: not original
> Follow the rules [CONTRIBUTING](https://github.com/vitejs/awesome-vite/blob/master/.github/contributing.md)

**Review by Scrum — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Silksofthesoul) [reply README.md]: @Scrum fixed

File: README.md [hunk 2]
```diff
@@ -135,6 +135,11 @@ Use the "Table on Contents" menu on the top-left corner to explore the list.
 - [vite-plugin-virtual-html](https://github.com/Windson1806/vite-plugin-virtual-html) - Make Vite MPA consistent with @vue/cli.
 - [vite-plugin-posthtml](https://github.com/chromeos/static-site-scaffold-modules/tree/main/modules/vite-plugin-posthtml) - Vite plugin to run [PostHTML](https://github.com/posthtml/posthtml). Currently only runs on served/built HTML files `index.html`.
 - [vite-plugin-plain-text](https://github.com/zheeeng/vite-plugin-plain-text) - A Vite plugin transforms the rule-matched file as plain text.
+- [vite-plugin-transform](https://github.com/Silksofthesoul/vite-plugin-transform) - Plugin resolve paths in cases where standard ways dont help, and replacement functionality.
+
+#### SSR
+
+- [vite-plugin-ssr](https://github.com/brillout/vite-plugin-ssr) - Like Nuxt/Next.js but with a simpler design that works with any view framework.
```
Comment (Scrum) [inline README.md:142]: this plugin already exists in https://github.com/vitejs/awesome-vite#libraries

**Review by Scrum — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Silksofthesoul) [reply README.md:142]: @Scrum I think I have no right to edit the information provided by other contributors. Somehow it happened that my changes are marked as having a conflict with the edits made earlier. I know how to categorize my plugin (the one and only one), but I'm not ready to think for other contributors. Therefore, I think I'd rather undo the current change, so as not to create bugs or offend someone by incorrectly categorizing their changes.

**Pull request closed without merging**